models/segmentation_template/docs/README.md

```diff
@@ -0,0 +1,75 @@
+
+# Template Segmentation Bundle
+
+This bundle is meant to be an example of segmentation in 3D which you can copy and modify to create your own bundle.
```

Comment:
A good template. Should we also modify the readme of the repo to mention it, thus users can see it directly?

Reply:
I've added a mention in the README, have a look but it's a section we can expand with new templates as we go.